core/src/main/java/org/neo4j/graphalgo/core/utils/ProgressLoggerAdapter.java

```diff
@@ -0,0 +1,30 @@
+package org.neo4j.graphalgo.core.utils;
+
+import org.neo4j.logging.Log;
+
+import java.util.function.DoubleSupplier;
+
+/**
+ * @author mknblch
+ */
+public class ProgressLoggerAdapter implements ProgressLogger {
+
+    private final Log log;
+
+    private int logInterval = 60_000;
```

Comment:
I would make this configurable and reduce it to 10s by default, but can do it in a later change.